src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -110,6 +115,25 @@ public void configure(Map<String, ?> configs) {
     private class StorageChecker implements Runnable {
         private final Thread storageCheckerThread = new Thread(this, "storage-quota-checker");
         private volatile boolean running = false;
+        private final Metric kafkaBrokerAllUsedBytes = Metrics.newGauge(mNameCreate("AllUsedBytes"), new Gauge<Long>() {
+            public Long value() {
+                return storageUsed.get();
+            }
+        });
+        private final Metric getKafkaBrokerSoftLimitBytes = Metrics.newGauge(mNameCreate("SoftLimitBytes"), new Gauge<Long>() {
+            public Long value() {
+                return storageQuotaSoft;
+            }
+        });
+
+        private MetricName mNameCreate(String name) {
+            MetricName metricName = new MetricName(StaticQuotaCallback.class, name);
+            return new MetricName(metricName.getGroup(),
+                    metricName.getType(),
+                    metricName.getName(),
+                    metricName.getScope(),
+                    metricName.getMBeanName().replace("\"", ""));
```

Comment:
I know this is for removing the quotes that I was talking about but I don't see it's used in the Kafka codebase. Why it's needed here? @tombentley any clue why Kafka exports metrics in MBean having names without quotes and here we need to remove them?

Reply:
@ppatierno Without the replace above by default when a metric name is created this way it is interpreted as a String object and is then fed into the pool as such. `com.yammer.metrics.core` is used to create metric names and expects String values to be fed into each parameter `(group, type, name, scope, mbeanname)`. I'm not entirely sure what is done differently on the kafka side but from what I've seen with experimenting through the plugin leveraging the Yammer API, it treats everything like a string object and will make it's own topology even if you were to put a pre-existing group in (ex. kafka.cluster would be "kafka.cluster" and exist alongside kafka.cluster in the exported JMX objects). Once the quotes are removed by the above method though the pre-existing group will just be added onto rather than another be created alongside.

Reply:
@tombentley so are you saying that Kafka itself doesn't use Yammer metrics? Or that we are just missing to use the utility sanitize method here? Sorry I didn't totally get your answer. I would like to understand the difference between a metric exported by Kafka itself (which doesn't get quotes) and a metric exported here (which gets quotes we need to remove).

Reply:
@ppatierno To answer this it looks like they do use the same method for calling the Yammer metrics. The reason the quotes were happening is because I didn't set an `mBeanName` when creating the Metric and so it looks like when that is the case the reporter places quotation marks around the object. You can use this method to obtain the `mBeanName` if you are unsure of it through this and then go back and fill in what the correct name should be. I pushed changes though so moving forward this shouldn't be an issue and custom metrics will just need to supply the name they want the metric to have.